ui/components/multichain/import-tokens-modal/import-tokens-modal.js

```diff
@@ -459,7 +459,9 @@ export const ImportTokensModal = ({ onClose }) => {
         setShowSymbolAndDecimals(false);
         break;
 
-      case Boolean(identities[standardAddress]):
+      case Boolean(
+        accounts.find((internalAccount) => internalAccount.address === address),
+      ):
```

Comment:
Why the change here to use the original address and not the standardized version?